.env.docker

```diff
@@ -0,0 +1,72 @@
+# Change this for production
+SECRET_KEY=ChangeMeToARandomString
+
+# Turn this to True for development.
+DEBUG=True
```

Comment:
Do we need to add a note about this being very dangerous to leave on in production? Not sure if it's still as dangerous as it used to be.

Reply:
The example template we give, `.env.example` has this False by default. This is True here because it's used by `docker-compose` to set up the tracker, but is not baked into the tracker image in any way. So it's safe for us to leave it like this.